pdns/recursordist/RECURSOR-MIB.txt

```diff
@@ -15,12 +15,15 @@ IMPORTS
         FROM SNMPv2-CONF;
 
 rec MODULE-IDENTITY
-    LAST-UPDATED "202408280000Z"
+    LAST-UPDATED "202504290000Z"
```

Comment:
(not for this PR really)

should this .txt be in git at all? it is generated

Reply:
We refer to it from docs, so it needs to be somewhere.